.github/workflows/ci_rust.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: cargo test --features unstable-renegotiate
 
       - name: Network-enabled integration tests
-        working-directory: ${{env.ROOT_PATH}}/integration
+        working-directory: ${{env.ROOT_PATH}}/standard/integration
```

Comment:
Are the s2n-tls-hyper tests still being run somewhere?

Reply:
good catch, adding them...